dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -53,8 +53,15 @@ def __init__(self, options=None):
         self.sum = 0
         self.variance = 0
         self.max_histogram_bin = 10000
-        self.histogram_bin_method_names = ['auto', 'fd', 'doane', 'scott',
-                                           'rice', 'sturges', 'sqrt']
+        if options is not None:
+            self.histogram_selection = options.histogram_and_quantiles.method
+        else:
+            self.histogram_selection = None
+        if self.histogram_selection is not None:
+            self.histogram_bin_method_names = [self.histogram_selection]
+        else:
+            self.histogram_bin_method_names = ['auto', 'fd', 'doane', 'scott',
```

Comment:
Is there an "all" option or something or do you explicitly state the list?

Reply:
So if nothing is specified via the option then it stays None, and then all are attempted as normal, if you want I can easily add support for an "all" keyword or something like that, but right now leaving it unspecified or `None` handles that.

Reply:
'all' or 'infer' may be more explicit than None

Reply:
typically to avoid the if else, what we've done in the past is set the defaults before the if statement and hten allow the if to change it if met, like:
```python
self.histogram_bin_method_names = ['auto', 'fd', 'doane', 'scott',
                                           'rice', 'sturges', 'sqrt']
self.histogram_selection = None
if options is not None and isinstance(options, NumericalOptions):
    self.histogram_selection = options.histogram_and_quantiles.method
    if self.histogram_selection:
        self.histogram_bin_method_names = [self.histogram_selection]
```

would need to double check it hits all edge cases